examples/foreign-toplevel.c

```diff
@@ -236,7 +242,7 @@ static void handle_global(void *data, struct wl_registry *registry,
 	} else if (strcmp(interface,
 			zwlr_foreign_toplevel_manager_v1_interface.name) == 0) {
 		toplevel_manager = wl_registry_bind(registry, name,
-				&zwlr_foreign_toplevel_manager_v1_interface, 1);
+				&zwlr_foreign_toplevel_manager_v1_interface, version);
```

Comment:
This is dangerous. For instance, if the compositor supports v3 and v3 happens to add a new event, this will make the client crash. Clients usually bind to `min(client_version, compositor_version)`.

Reply:
I did this here since the example client will always be updated to support the compositor version?

Reply:
But if we want to make this copy-paste-proof I get your point

Reply:
Yeah, I'd rather hardcode it in the client if we want to only support the latest version

Reply:
well it requires the latest version, so I'll just hardcode 2 for now